aten/src/ATen/native/tags.yaml

```diff
@@ -3,6 +3,15 @@
 - tag: inplace_view
   desc: |
           This tag indicates if an operator *only* modifies the tensor metadata
+- tag: pt2_compliant_tag
```

Comment:
torch.Tag.pt2_compliant_tag and at::Tag::pt2_compliant_tag may sound a bit goofy, but they are easily greppable